book/functors/README.md

```diff
@@ -177,12 +177,12 @@ primitives for interacting with intervals:
       if Endpoint.compare low high > 0 then Empty
       else Interval (low,high)
 
-    (** Returns true iff the interval is empty *)
```

Comment:
Not a typo: http://mathworld.wolfram.com/Iff.html

(though arguably should be changed to "if and only if" explicitly.)